### DISCUSSION TITLE
Recommended best practices when installing WebView2 on Citrix?

### DISCUSSION THREAD

**Discussion by shamvaghmare** [MicrosoftEdge/WebView2Feedback#3948] [Q&A]
We are facing issues that WebView2 control in Winforms application renders the dialog blank or gray.
Is there best practice recommended by Microsoft when installing WebView2 in Citrix environment ?

**Comment by champnic** [comment]
Hey @shamvaghmare - if you search the issues the "Citrix" there are other threads that might be similar. Can you see if one of those threads answers your question? If not let me know and we can open this as a bug.
